pkg/gardenlet/controller/seed/seed/components.go

```diff
@@ -370,6 +370,47 @@ func (r *Reconciler) newIstio(ctx context.Context, seed *seedpkg.Seed, isGardenC
 		}
 	}
 
+	// Allow temporary creation of ingress gateway copy to easy migration
```

Comment:
Can you add a TODO statement to make sure this code block is removed again? Or do you plan to support it for a longer period of time?

Reply:
Good point. Added with https://github.com/gardener/gardener/pull/9304/commits/d150c101eb3a753f59da3f202b9532741c400c3f.